datasets/ONS-UK-Trade-in-goods-CPA-08/main.py

```diff
@@ -5,43 +5,36 @@
 #       extension: .py
 #       format_name: light
 #       format_version: '1.5'
-#       jupytext_version: 1.13.0
+#       jupytext_version: 1.13.8
 #   kernelspec:
-#     display_name: Python 3 (ipykernel)
+#     display_name: Python 3.9.12 64-bit
 #     language: python
 #     name: python3
 # ---
 
 # UK trade in goods, CPA (08)
 
-# +
-import json
 import pandas as pandas
 from gssutils import *
 
-pd.options.mode.chained_assignment = None 
-pd.set_option('display.float_format', lambda x: '%.0f' % x) 
-
-cubes = Cubes('info.json')
-info = json.load(open('info.json'))
-landingPage = info['landingPage']
 metadata = Scraper(seed="info.json")
+
 distribution = metadata.distribution(latest = True)
+
 title = distribution.title
-distribution
-# -
 
 tabs = distribution.as_databaker()
-tidied_sheets = []
 
+tidied_sheets = []
 for tab in tabs:
-    if tab.name not in ['Index', 'Contact']:
+    if tab.name not in ['Index', 'Contact', 'Table of contents', 'Cover sheet', 'Notes']:
```

Comment:
You could remove "Index" and "Contact" as the tab name doesn't exist

Reply:
Oh thanks for spotting that and would adjust it asap